src/pages/home/Home.tsx

```diff
@@ -1,12 +1,15 @@
-import { MOCKED_STUDENTS } from "../../mocks/students";
 import "./Home.css";
 import infoIcon from "../../assets/info-icon.webp";
 import trashIcon from "../../assets/trash-icon.webp";
 import activateIcon from "../../assets/activate-icon.svg";
-import { useNavigate } from 'react-router';
-function Home() {
+import { useNavigate } from "react-router";
+import { useGetUserStudentsQuery } from "../../app/services/UserService";
+import useAuthentication from "../../hooks/useAuthentication";
 
-  const navigate = useNavigate()
+function Home() {
+  const { userId } = useAuthentication();
+  const { data } = useGetUserStudentsQuery(userId!);
```

Comment:
Los hooks que nos generan RTK Query cuando hacemos un `query` (no una mutation), nos devuelve un objeto con diferentes cosas:
- data
- isLoading
- isError

Son diferentes atributos que nos permiten utilizarlos para poder manejar la logica en nuestra pagina (ejemplo, mostrar un componente `<Loading>` cuando el isLoading esta en true). Ahora, si bien el hook nos devuelve la informacion la peticion http en un atributo llamado `data`, esto es muy generico... ¿qué tiene data, estudiantes, metodos de pago, usuarios? Para que sea más declarativo podríamos hacer lo siguiente


```suggestion
  const { data: students } = useGetUserStudentsQuery(userId!);
```

Estariamos transformando el data en `students`, y de esta manera cuando lo utilicemos, sabemos que ese atributo tiene estudiantes